### PR TITLE
feat(broker): request created topics over the management api

### LINKED ISSUE #1005: A broker can request the partition ids of a topic
AT:
* a broker can request the partition ids of a topic via management api

Context:
* when the workflow instance arrives a message catch event then it opens a subscription on one partition of the current topic
* the partition is chosen by a hash function
* in order to select the partition, the processor needs to know all existing partitions of the topic

### PR DESCRIPTION
**Pull request by saig0**
* a broker can request the partition ids of a topic via management api

closes #1005

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 305f1f7d7b391acffb6cdaafc7cd5fe39292dc42
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, ci**

Guidelines are available at https://github.com/zeebe-io/zeebe/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

File: broker-core/src/main/java/io/zeebe/broker/system/management/topics/FetchCreatedTopicsRequest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.zeebe.broker.system.management.topics;
+
+import io.zeebe.broker.util.SbeBufferWriterReader;
+import io.zeebe.clustering.management.FetchCreatedTopicsRequestDecoder;
+import io.zeebe.clustering.management.FetchCreatedTopicsRequestEncoder;
+
+public class FetchCreatedTopicsRequest
```
Comment (ThorbenLindhauer) [inline broker-core/src/main/java/io/zeebe/broker/system/management/topics/FetchCreatedTopicsRequest.java]: This class is never used.

File: broker-core/src/main/java/io/zeebe/broker/system/management/topics/FetchCreatedTopicsRequestHandlerService.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+package io.zeebe.broker.system.management.topics;
+
+import io.zeebe.broker.clustering.base.partitions.Partition;
+import io.zeebe.broker.clustering.orchestration.state.KnownTopics;
+import io.zeebe.broker.system.management.LeaderManagementRequestHandler;
+import io.zeebe.servicecontainer.Injector;
+import io.zeebe.servicecontainer.Service;
+import io.zeebe.servicecontainer.ServiceGroupReference;
+
+public class FetchCreatedTopicsRequestHandlerService
```
Comment (ThorbenLindhauer) [inline broker-core/src/main/java/io/zeebe/broker/system/management/topics/FetchCreatedTopicsRequestHandlerService.java]: This looks unnecessarily complex. Why have an extra service and not just merge it with `LeaderManagementRequestHandler`?

**Review by ThorbenLindhauer — CHANGES_REQUESTED** [review]
* see notes at change set
* There are no tests. With the goal of a stable `develop` branch, we should not have untested (and without tests also unused) code. If you disagree, we should probably discuss this in the team.

**Comment by ThorbenLindhauer** [comment]
In addition, commit https://github.com/zeebe-io/zeebe/pull/1031/commits/81537188c4de9d2dca3f3be8983bb3e0c99d4750 should be squashed into the previous commit.

**Comment by saig0** [comment]
@ThorbenLindhauer , I added a test case. I'll squash the commits after the review is done.

**Review by ThorbenLindhauer — APPROVED** [review]

**Pull request merged**